modules/programs/pdf2md/main.py

```
@@ -0,0 +1,235 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)